### PR TITLE
Completed exercise

### PR DESCRIPTION
**Pull request by Lockless404**
Created myEnumerable module with all? , any? and filter?
Created myList class with each and initialize methods to create an array.

### CODE REVIEW

**Review by ZeenatLawal — COMMENTED** [review]
Hi <STUDENT_MENTION>, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ 

You have submitted a project, that doesn't have a descriptive README file. Kindly write a descriptive README with all instructions following the Microverse template.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

File: my_list.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+class MyList
+  @list = []
+  include MyEnumerable
```
Comment (bini-i) [inline my_list.rb:5]: - [ ] You have done a great job including MyEnumerable module in your class, but kindly make sure the file is available(required) in your current file before including it.

**Review by bini-i — CHANGES_REQUESTED** [review]
# Status: Changes Required :yellow_circle:            
### Hello @Lockless404  :wave: 

Your project looks good and shows you have put a lot of effort into it. Kindly work on the following suggestions to further improve your project.
## To highlight
- All the methods are implemented very nicely and works perfectly
- Correct linter and Gitflow setup
## Optional suggestions
-   [_OPTIONAL_]

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

#### If you have any questions feel free to ask within the PR thread.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
### HAPPY CODING :computer: :blush: 

> #### Reviewed by Binyam Hailemeskel (Code Reviewer)
> GitHub: [@bini-i](https://github.com/bini-i) 
> Twitter: [@binyamshewa](https://twitter.com/binyamshewa)
> LinkedIn: [LinkedIn](https://www.linkedin.com/in/binyam-hailemeskel-728048151/)

**Review by enaburekhan — APPROVED** [review]
Hi Lockless404, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Code review by **Eric Enaburekhan**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**